Private/Script.Configuration.ps1

```diff
@@ -68,7 +68,7 @@ $Script:Configuration = [ordered] @{
             Header       = @{
                 CssLink = 'https://cdn.datatables.net/v/dt/jq-3.3.1/dt-1.10.18/af-2.3.2/b-1.5.4/b-colvis-1.5.4/b-html5-1.5.4/b-print-1.5.4/cr-1.5.0/fc-3.2.5/fh-3.1.4/kt-2.5.0/r-2.2.2/rg-1.1.0/rr-1.2.4/sc-1.5.0/sl-1.2.6/datatables.min.css'
                 Css     = "$PSScriptRoot\..\Resources\CSS\datatables.min.css"
-                JsLink  = "https://cdn.datatables.net/v/dt/jq-3.3.1/dt-1.10.18/af-2.3.2/b-1.5.4/b-colvis-1.5.4/b-html5-1.5.4/b-print-1.5.4/cr-1.5.0/fc-3.2.5/fh-3.1.4/kt-2.5.0/r-2.2.2/rg-1.1.0/rr-1.2.4/sc-1.5.0/sl-1.2.6/datatables.min.js"
+                JsLink  = "https://cdn.datatables.net/v/dt/jq-3.3.1/dt-1.10.18/af-2.3.2/b-1.5.4/b-colvis-1.5.4/b-html5-1.5.4/b-print-1.5.4/cr-1.5.0/fc-3.2.5/fh-3.1.4/kt-2.5.0/r-2.2.2/rg-1.1.0/rr-1.2.4/sc-1.5.0/sl-1.2.6/datatables.min.js","https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.8.4/moment.min.js","https://cdn.datatables.net/plug-ins/1.10.19/sorting/datetime-moment.js"
```

Comment:
Do you need to have both? moment.min.js and datetime-moment.js? If yes, that's ok. Can you also download those files and put in "$PSScriptRoot\..\Resources\JS\datatables.min.js". JSLink is used by default, but people can choose to integrate code into HTML. Not advised but still an option.

Reply:
yes both neede.
Ah yes, I didn't recognize the possibility for local ressources :)

Will fix it.